packages/app/src/app/overmind/namespaces/comments/state.ts

```diff
@@ -2,8 +2,9 @@ import { CommentsFilterOption } from '@codesandbox/common/lib/types';
 import { CommentFragment, CommentWithRepliesFragment } from 'app/graphql/types';
 import { Derive } from 'app/overmind';
 import isToday from 'date-fns/isToday';
+import { v4 as uuidv4 } from 'uuid';
 
-export const OPTIMISTIC_COMMENT_ID = 'OptimisticCommentId';
+export const OPTIMISTIC_COMMENT_ID = uuidv4();
```

Comment:
Ops, this needs to be a factory or we will have the same ID on every new comment :-)

`export const OPTIMISTIC_COMMENT_ID = () => uuidv4();`

But! I would rather remove this and use `uuid` directly wherever we need to create an id.

That way we can also remove any logic related to `OPTIMISTIC_COMMENT_ID` in components, cause it is actually not needed anymore as we always have an ID to reference now 😄